docs/vendor/anthropic/agent-sdk/python/REFERENCE.md

```
@@ -0,0 +1,2098 @@
+
+Complete API reference for the Python Agent SDK, including all functions, types, and classes exposed by `claude_agent_sdk`.
+
+## Installation
+
+```bash
+pip install claude-agent-sdk
+```
+
+| Property | Type | Default | Description |
+| :-- | :-- | :-- | :-- |
+| `tools` | `list[str] | ToolsPreset | None` | `None` | Base tool set passed to the CLI (`--tools`). Use `{"type": "preset", "preset": "claude_code"}` for default toolset. |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)